apps/masterbots.ai/lib/hooks/use-mb-chat.ts

```
@@ -130,7 +127,25 @@ export function useMBChat(config?: MBChatHookConfig): MBChatHookCallback {
+    async onFinish(message, options) {
+      setLoadingState(undefined)
+      setActiveTool(undefined)
+      setIsNewResponse(false)
+
+      if (options.finishReason === 'error') {
+        toast.error('Failed to send message. Please try again.')
+
+        if (isNewChat) {
+          await deleteThread({
+            threadId: params?.threadId ?? activeThread?.threadId,
+            jwt: session!.user?.hasuraJwt,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)